pkg/controllers/webhook/controller.go

```diff
@@ -431,6 +431,7 @@ func (c *controller) updatePolicyStatuses(ctx context.Context) error {
 		status := policy.GetStatus()
 		status.SetReady(ready)
 		status.Autogen.Rules = nil
+		setRuleCount(policy, status)
```

Comment:
We're going to call `ComputeRules` twice here, can we reduce the number of calls to one ?